framework/src/main/java/org/checkerframework/framework/util/AtmLubVisitor.java

```diff
@@ -91,7 +91,7 @@ private AnnotatedTypeMirror lubWithNull(
     } else {
       otherAsLub = AnnotatedTypes.asSuper(atypeFactory, otherType, lub);
     }
-    TypeMirror otherTM = otherAsLub.getUnderlyingType();
+    TypeMirror otherTM = otherType.getUnderlyingType();
```

Comment:
Is this logically part of this pull request?  Should it be committed in a separate PR?